python/tests/test_variable.py

```diff
@@ -1010,3 +1010,27 @@ def test_equal():
 def test_not_equal():
     var = sc.Variable()
     assert_export(sc.not_equal, x=var, y=var)
+
+
+def test_radd_int():
+    var = sc.Variable(dims=['x'], values=[1,2,3])
+    assert (var + 1).dtype == sc.dtype.int64
```

Comment:
I remember Python ints are different on different platforms (32 bit on Windows?), but maybe that was an older Python? See also comment below.